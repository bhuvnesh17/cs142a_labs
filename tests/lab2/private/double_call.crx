// syntax error

func main() : type {
  ::::();
}
