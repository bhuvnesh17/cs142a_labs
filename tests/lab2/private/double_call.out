Error parsing file.
SyntaxError(4,5)[Expected IDENTIFIER but got CALL.]
SyntaxError(4,5)[Could not complete parsing.]
